packages/backend-modules/access/lib/memberships.js

```diff
@@ -21,6 +21,22 @@ const addRole = async (grant, user, pgdb, role) => {
   return false
 }
 
+const removeRole = async (grant, user, pgdb, role) => {
```

Comment:
`addRole` was a helper function for `addMemberRole` and later exposed to avoid code repetation.

Would implement `removeRole` into [`removeMemberRole`](https://github.com/republik/plattform/pull/113/files#diff-86d6bc219d3e87a6d49a689923f741e568d7466771b83993bbb83a3c39edd1c4R62-R75) now, too.